### ISSUE TITLE
Improve generation of TemplateExtension with code action

### ISSUE THREAD

**Issue by angelozerr** [redhat-developer/quarkus-ls#681]
Given this Qute template:

{book.XXXX}

The code action which generate the template extension method in a new class will look like this:

```java
@io.quarkus.qute.TemplateExtension
public class TemplateExtensions {
	public static String XXXX(org.agoncal.article.qutepanache.Book book) {
		return null;
	}
}
```

We should improve:

 * generate a package but which strategy can ve adopt it to get a package from sources ? In the future perhaps we should have a popup to fill the package + class name of the template extension.
 * add a line space after the TemplateExtensions 
```java
public class TemplateExtensions {

	public static String XXXX(org.agoncal.article.qutepanache.Book book) {
```
 * generate import of the Book page to avoid using full qualified name in the parameter.

In other words:

```java
package org.acme;

import org.agoncal.article.qutepanache.Book;

@io.quarkus.qute.TemplateExtension
public class TemplateExtensions {

	public static String XXXX(Book book) {
		return null;
	}
}
```
